test/common/vmmanage.py

```diff
@@ -30,22 +30,23 @@ def upload_scripts(machine, args):
     if args["containers"]:
         machine.upload([os.path.join(testinfra.TEST_DIR,"..", "containers")], "/var/tmp")
 
-def run_install_script(machine, do_build, do_install, skip, arg, args):
+def run_install_script(machine, do_build, do_install, skips, arg, args):
     install = do_install
     if args["containers"]:
         do_install = False
 
+    skip_args = map(lambda skip: " --skip '%s'" % skip, skips)
     machine.execute("cd /var/tmp; ./%s.install%s%s%s%s%s%s" % (machine.image,
                                                          " --verbose" if args["verbose"] else "",
                                                          " --quick" if args["quick"] else "",
                                                          " --build" if do_build else "",
                                                          " --install" if do_install else "",
-                                                         " --skip '%s'" % skip if skip else "",
+                                                         " ".join(skip_args),
                                                          " '%s'" % arg if arg else ""))
     if install and args["containers"]:
         machine.execute("/var/lib/testvm/containers.install")
 
-def build_and_maybe_install(image, do_install=False, skip=None, args=None):
+def build_and_maybe_install(image, do_install=False, skips=[], args=None):
```

Comment:
Mutable types as a default argument for a kwarg are usually a bad idea. Better to use None and instantiate to a blank value in the code.